website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,189 @@
+
+<details>
+<summary>What are linters and formatters? </summary>
+Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules. 
+</details>
+
+
+In the dbt Cloud IDE, you have the capability to perform linting, auto-fix, and formatting on five different file types:
+ 
+- SQL &mdash; [Lint](#lint) and fix with SQLFluff, and [format](#format) with sqlfmt
+- YAML, Markdown, and JSON &mdash; Format with Prettier
+- Python &mdash; Format with Black
+
+In the dbt Cloud IDE, each file type has its own unique linting and formatting rules. You have the option to [customize](#customize-linting) the linting process to add more flexibility and enhance problem/style detection. 
+
+By default, the IDE uses sqlfmt rules to format your code, making it convenient to use right away. However, if you have a file named `.sqlfluff` in the root directory of your dbt project, the IDE will default to SQLFluff rules instead.
+
+<DocCarousel slidesPerView={1}>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/sqlfluff.gif" width="100%" title="Use SQLFluff to lint/format your SQL code, and view code errors in the Code Quality tab."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/sqlfmt.gif" width="95%" title="Use sqlfmt to format your SQL code."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/prettier.gif" width="95%" title="Format YAML, Markdown, and JSON files using Prettier."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-sql-popup.jpg" width="95%" title="Use the Config button to select your tool."/>
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-sqlfluff-config.jpg" width="95%" title="Customize linting by configuring your own linting code rules, including dbtonic linting/styling."/>
+
+</DocCarousel>
+
+## Lint
+
+With the dbt Cloud IDE, you can seamlessly use [SQLFluff](https://sqlfluff.com/), a configurable SQL linter, to warn you of complex functions, syntax, formatting, and compilation errors. This integration allows you to run checks, fix, and display any code errors directly within the Cloud IDE:
+
+- Works with Jinja and SQL, 
+- Comes with built-in [linting rules](https://docs.sqlfluff.com/en/stable/rules.html). You can also [customize](#customize-linting) your own linting rules.
+- Empowers you to [enable linting](#enable-linting) with options like **Lint** (displays linting errors and recommends actions) or **Fix** (auto-fixes errors in the IDE).
+- Displays a **Code Quality** tab to view code errors, and provides code quality visibility and management. 
+
+### Enable linting
+
+1. To enable linting, make sure you've opened a `.sql` file.
+2. Click on the **`</> Config`** button on the right side of the console. 
```

Comment:
Which console? IDE? Will this option only appear if it's a SQL file or will it persist regardless of the file type?
```suggestion
2. Click on the **`</> Config`** button on the right side of the IDE. 
```

Reply:
only if it's a sql file, exactly why i asked this question [here](https://dbt-labs.slack.com/archives/C012TTNN0JK/p1685707525554349?thread_ts=1685704915.113989&cid=C012TTNN0JK) last week and why i was trying to be extra clear that they need to be in a sql file